core/risk/engine.go

```diff
@@ -188,7 +194,7 @@ func (e *Engine) UpdateMarginAuction(ctx context.Context, evts []events.Margin,
 	// for now, we can assume a single asset for all events
 	rFactors := *e.factors
 	for _, evt := range evts {
-		levels := e.calculateAuctionMargins(evt, price, rFactors)
+		levels := e.calculateAuctionMargins(evt, price, rFactors, e.linearSlippageFactor, e.quadraticSlippageFactor)
```

Comment:
I could be wrong about this, but do we have to pass `e.linearSlippageFactor, e.quadraticSlippageFactor` here, if they are accessible through the receiver? Can these parameters have different values?

Reply:
you're right but if you don't mind I'll fix it in a followup pr